.gitignore

```diff
@@ -7,6 +7,8 @@
 /.bundle
 /db/*.sqlite3
 /db/*.sqlite3-journal
+/db/schema.rb
+db/seeds.rb
```

Comment:
Just noticed, these files should be in .gitignore
Will add a commit to remove it from here. Don't worry about it.